idl/chromadb/proto/coordinator.proto

```diff
@@ -408,7 +409,7 @@ message ListCollectionsToGcRequest {
   // before this cutoff time.
   // SysDb can apply additional logic to return the collections that should
   // be prioritized for GC.
-  optional uint64 cutoff_time_secs = 1;
+  optional google.protobuf.Timestamp cutoff_time = 1;
```

Comment:
we don't care about backwards compatibility 
(I figured a timestamp type works better as a way of explicitly communicating that this is an absolute timestamp)